src/App.ts

```diff
@@ -1591,9 +1608,11 @@ function isBlockActionOrInteractiveMessageBody(
   return (body as SlackActionMiddlewareArgs<BlockAction | InteractiveMessage>['body']).actions !== undefined;
 }
 
-// Returns either a bot token or a user token for client, say()
+/** 
+ * Returns a bot token, bot access token or user token for client, say()
```

Comment:
Per comment above: When a botAccessToken exists on the context, we prefer over regular xoxb.